pkg/api/swagger_responses.go

```diff
@@ -81,3 +81,131 @@ type UnauthorizedError GenericError
 //
 // swagger:response acceptedResponse
 type AcceptedResponse GenericError
+
+// documentation for PublicError defined in errutil.Error
+
+// swagger:response publicErrorResponse
+type PublicErrorResponse struct {
+	// The response message
+	// in: body
+	Body PublicError `json:"body"`
+}
+
+// PublicError is derived from Error and only contains information
+// available to the end user.
+// swagger:model publicError
+type PublicError struct {
```

Comment:
I could use [this struct ](https://github.com/grafana/grafana/blob/main/pkg/util/errutil/errors.go#L371) but adding swagger metadata to that file won't work when creating the swagger specification.

Reply:
can you provide more details what exactly didn't work?

Reply:
I created [this draft PR](https://github.com/grafana/grafana/pull/75592) with the refactor and the differences I got on file spec generation.

Reply:
Great I will change them by the generic bad request response